subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishCustomComponentIntegTest.groovy

```diff
@@ -112,6 +111,14 @@ class MavenPublishCustomComponentIntegTest extends AbstractMavenPublishIntegTest
                     maven { url "${mavenRepo.uri}" }
                 }
             }
+            
+            class TestAttributes {
+                // shared mutable state for tests, don't do this at home!
```

Comment:
Why can't we use `AttributeTestUtil` here?

Reply:
AttributeTestUtil can be used in the test setup itself, but not within the execution of a test script.

Reply:
Of course. I was misreading the test. Thanks for explaining.